<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчет налога</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="8">
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>Печать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Дата С</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>По</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="de_date_from">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDateEdit" name="de_date_to">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>Расчитать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Экспорт</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Налог</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="le_nalog">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>870</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Принят</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Уволен</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дней</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Налог</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(89, 125, 163);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>-90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>-49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_calc()</slot>
  <slot>ui_print()</slot>
  <slot>ui_export()</slot>
 </slots>
</ui>
